channels/ON-CHAIN.md

```diff
@@ -85,6 +86,23 @@ channel_id = Blake2b(initiator || channel_create_tx_nonce || responder)
                         32                  32                  32
 ```
 
+Each party keeps a state tree specific for the channel. It consists of all the
+channel data: accounts, contracts and etc. Closing transactions provide a
```

Comment:
I understand these "closing transactions" are channel_close_solo and channel_slash. Ok.